app.yaml

```diff
@@ -62,6 +62,10 @@ handlers:
   # TODO(Sean Lip): Add cache when system to break cache during
   # new release is figured out.
     Cache-Control: 'no-cache'
+- url: /extensions
+  static_dir: extensions
+  application_readable: true
```

Comment:
Right now it's required since we require some py scripts from that directory. But, we will remove it when we are serving all resources from a build directory.